packages/web-twig/src/Resources/components/Accordion/AccordionContent.twig

```diff
@@ -0,0 +1,35 @@
+{# API #}
+{%- set props = props | default([]) -%}
+{%- set _class = props.class | default(null) -%}
+{%- set _id = props.id -%}
+{%- set _labelId = props.labelId -%}
```

Comment:
I think we should use a better name here. It looks like we use the `<label>` tag and we are setting an id for this. Maybe `labelledById` will be better or `ariaLabelledBy`...

Reply:
ok, renamed